docker/checkout.bash

```diff
@@ -40,7 +40,7 @@ done
 cd "${dir}"/src
 
 git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-msgs.git --branch "${BRANCH}"
-git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-utils.git --branch "${BRANCH}"
+git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-utils.git --branch arc-146-remove-ros1
```

Comment:
This can be reverted to the argument

Reply:
I can remove it before I merge, since the build will fail